### PR TITLE
Make DocumentPropertyType extend GMLPropertyType

### PR DESCRIPTION
**Pull request by frankfu03**

### CODE REVIEW

File: src/test/resources/MOBS.xml [hunk 1]
```diff
@@ -221,22 +221,24 @@
...
-            <geo:foundationDepth>4.9</geo:foundationDepth>
-            <geo:markerDescription>Stainless Steel Plate</geo:markerDescription>
-            <geo:dateInstalled>2002-09-15Z</geo:dateInstalled>
-            <geo:geologicCharacteristic>Thin Bedrock of between 5-20m of</geo:geologicCharacteristic>
-            <geo:bedrockType>SANDSTONE</geo:bedrockType>
-            <geo:bedrockCondition>Firm, variably oxidised, variably cemented</geo:bedrockCondition>
-            <geo:fractureSpacing>No Fractures</geo:fractureSpacing>
-            <geo:faultZonesNearby>Non documented</geo:faultZonesNearby>
-            <geo:notes>antenna mounting plate fixed to 1.5m stainless steel pole. Pole is fixed to 0.6m diameter bluestone pillar The GPS antenna is located on top of a stainless steel pillar, which has been secured into the top of a bluestone pillar set in bedrock</geo:notes>
+            <geo:SiteIdentification>
```
Comment (lbodor) [inline src/test/resources/MOBS.xml]: missing gml:id

**Review by lbodor — COMMENTED** [review]
There should be one logical change per PR.

File: src/test/resources/MOBS.xml (hunk 1 above)
Comment (frankfu03) [reply src/test/resources/MOBS.xml]: Yes, gml:id was added. Thanks.

**Comment by frankfu03** [comment]
Yeah, I have split it into two PRs.

**Review by lbodor — APPROVED** [review]

**Pull request merged**